examples/quickstart/quickstart.ipynb

```diff
@@ -63,7 +63,8 @@
    "outputs": [],
    "source": [
     "# Install the ZenML CLI tool and Tensorflow\n",
-    "!pip install zenml tensorflow"
+    "!pip install zenml \n",
+    "!zenml integration install tensorflow"
```

Comment:
I think tensorflow is already installed in colabs at least but its fine to keep it in

Reply:
I think some users will run this in a jupyter notebook within their own environments.

Reply:
Yeah (Y)